pkgs/development/python-modules/torcheval/default.nix

```diff
@@ -0,0 +1,112 @@
+{
+  lib,
+  callPackage,
+  buildPythonPackage,
+  fetchFromGitHub,
+
+  # build-system
+  setuptools,
+
+  # dependencies
+  typing-extensions,
+
+  # tests
+  numpy,
+  pytestCheckHook,
+  pytest-timeout,
+  scikit-learn,
+  torchvision,
+  cython_0,
+}:
+let
+  pname = "torcheval";
+  version = "0.0.6";
+
+  # The torcheval 0.0.6 lib depends on a torchtnt>=0.0.5, however the available versions
+  # of torchtnt on nixpkgs (0.4.2 at the time of writing) are not compatible due to missing methods.
+  #
+  # To remedy this, the torchtnt-nightly commit on Github that was published on the same day as
+  # the torcheval 0.0.6 lib is used in lieu of a properly packaged torchtnt lib,
+  # as it is doubtful that this particular release of torchtnt is useful outside of this package.
+  torchtnt = callPackage ./torchtnt-nightly.nix { };
```

Comment:
We should work with overrides if possible or add a second version. Private libraries like this are not that great

Reply:
Ahh yes, an override would be perfect here! I'll follow up on this, thanks for the suggestion!